client/app/components/advisory-message-bar.js

```diff
@@ -0,0 +1,11 @@
+import Component from '@ember/component';
+import { tracked } from '@glimmer/tracking';
+import ENV from 'labs-zap-search/config/environment';
+
+export default class AdvisoryMessageBarComponent extends Component {
+  @tracked
+  showSandboxWarningOn = ENV.featureFlagShowSandboxWarning;
+
+  // @argument
+  message = false;
```

Comment:
Maybe make this an empty string if you need a default value? That way you aren't mixing bools and strings on the same property.